README.md

```diff
@@ -45,10 +45,10 @@ export BRANCH_ENV=master # or rc if branching from rc
 # export any custom external API URLs by editing env/custom.sh
 yarn run start
 ```
-
 Example pages:
  - http://localhost:3000/
  - http://localhost:3000/patient?studyId=lgg_ucsf_2014&caseId=P04
+> **Tip:** If you see dependency errors, especially the error that the script cannot identifying the packages managed by lerna(monorepo), you could do a `yarn build` first before starting the project.
```

Comment:
Running `yarn buildModules` alone may fix the issue and it is faster than `yarn build`, so maybe a tip like this is better?

```suggestion
> **Tip:** If you see dependency errors, especially the error that the script cannot identify the packages managed by lerna(monorepo), you could do a `yarn build` or `yarn buildModules` first before starting the project.
```

Reply:
@onursumer let's just include the `yarn buildModules` then, updated.